packages/components/src/input-group/tests/chromatic/InputGroup.chroma.jsx

```diff
@@ -0,0 +1,975 @@
+import { Autocomplete } from "@components/autocomplete";
+import { Button, IconButton } from "@components/button";
+import { DateInput, DateRangeInput } from "@components/date-input";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong